packages/block-library/src/image/index.php

```diff
@@ -11,9 +11,25 @@
  *
  * @param  array  $attributes The block attributes.
  * @param  string $content    The block content.
- * @return string Returns the block content with the data-id attribute added.
+ * @return string Returns the block content with the data-id attribute and aria-describedby added.
  */
 function render_block_core_image( $attributes, $content ) {
+
+	// If an image block has no alternative text but has a caption,
+	// and aria-describedby is not set, add aria-describedby to the image or image link.
+	if ( empty( $attributes['alt'] ) &&
```

Comment:
This is not working because `$attributes['alt']` is always empty, even if the user has added a text. I am testing other solutions.